ndb/src/google/cloud/ndb/eventloop.py

```diff
@@ -89,52 +78,61 @@ def clear(self):
             idlers = self.idlers
             queue = self.queue
             rpcs = self.rpcs
-            _logging_debug('Clearing stale EventLoop instance...')
+            _logging_debug("Clearing stale EventLoop instance...")
             if current:
-                _logging_debug('  current = %s', current)
+                _logging_debug("  current = %s", current)
             if idlers:
-                _logging_debug('  idlers = %s', idlers)
+                _logging_debug("  idlers = %s", idlers)
             if queue:
-                _logging_debug('  queue = %s', queue)
+                _logging_debug("  queue = %s", queue)
             if rpcs:
-                _logging_debug('  rpcs = %s', rpcs)
-            self.__init__(self.clock)
+                _logging_debug("  rpcs = %s", rpcs)
+            self.__init__()
             current.clear()
             idlers.clear()
             queue[:] = []
             rpcs.clear()
-            _logging_debug('Cleared')
+            _logging_debug("Cleared")
 
     def insort_event_right(self, event):
-        """Insert event in queue, and keep it sorted by `event.when` assuming
-        queue is sorted.
+        """Insert event in queue with sorting.
+
+        This function assumes the queue is already sorted by ``event.when`` and
+        inserts ``event`` in the queue, maintaining the sort.
 
         For events with same `event.when`, new events are inserted to the
         right, to keep FIFO order.
 
         Args:
-          event: a (time in sec since unix epoch, callback, args, kwargs)
-          tuple.
+            event (_Event): The event to insert.
         """
         queue = self.queue
-        lo = 0
-        hi = len(queue)
-        while lo < hi:
-            mid = (lo + hi) // 2
+        low = 0
+        high = len(queue)
+        while low < high:
+            mid = (low + high) // 2
             if event.when < queue[mid].when:
-                hi = mid
+                high = mid
             else:
-                lo = mid + 1
-        queue.insert(lo, event)
+                low = mid + 1
+        queue.insert(low, event)
 
     def queue_call(self, delay, callback, *args, **kwargs):
-        """Schedule a function call at a specific time in the future."""
+        """Schedule a function call at a specific time in the future.
+
+        Arguments:
+            delay (number): Time in seconds to delay running the callback.
```

Comment:
`number` not a valid type? How does this render when you run `nox -s docs`?

Instead `Union[int, float]` or similar?

Reply:
I figured a human reader would know which were the numeric types in Python. The style guide doesn't give any real guidance for cases where it just needs to be numeric.  I could just say `float` or `Union[int, float]` if that is more style conformant.

Reply:
e981c0c